build.gradle

```diff
@@ -111,6 +111,11 @@ project.afterEvaluate {
                     <option name="lenBefore" value="2" />
                 </LanguageOptions>
                 """.stripIndent()))
+            copyrightManager.append(new XmlParser().parseText("""
```

Comment:
There is probably no need to call append()/parseText twice -- it seems like you could add that piece of XML inside the string above.

But other than that, I checked out this branch locally and it works. Thanks!